lib/generators/templates/factory_bot/dev.rake

```diff
@@ -0,0 +1,12 @@
+if Rails.env.development? || Rails.env.test?
+  require "factory_bot"
+
+  namespace :dev do
+    desc "Sample data for local development environment"
+    task prime: "db:setup" do
+      include FactoryBot::Syntax::Methods
+
+      # create(:user, email: "user@example.com", password: "password")
+    end
+  end
+end
```

Comment:
Is this still an organizational convention? Would now be a good time to add this to the `./bin/setup` script too?

Reply:
Here’s some [context](https://thoughtbot.com/blog/priming-the-pump) as to why we use this.

Maybe now is a good opportunity to update the task description and link to the article.

Reply:
I wouldn't expect the `dev:prime` task to come from the `factory_bot` generator, as you could very well have dev data built through a different mechanism. If it does exist, though, it would be nice to add the FactoryBot helper methods.

Reply:
I agree that this needs more thought. I was porting this over from the existing generator, but wonder if now is a good opportunity to re-think this.

For example, on my client project, we rely solely on the `./bin/setup` script, which simply uses `db/seeds.rb`. That file looks something like this:

```rb
require "factory_bot_rails"
require "webmock"

class Seeder
  include FactoryBot::Syntax::Methods

 def cal
  # create :user
 end

Seeder.new.call
```

Reply:
> The handling of development data feels out of scope in this pull request.

I agree. It feels like we could have a separate generator for development data as well as an improved `./bin/setup` script.

Reply:
62eccbe0bd86318eb22d9f5ddb50c6bc384e4ad1